Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/Ltc.azsli

```diff
@@ -80,14 +80,44 @@ float3x3 BuildViewAlignedOrthonormalBasis(in float3 normal, in float3 dirToView)
 // xy plane in positive z space.
 float IntegrateEdge(float3 v1, float3 v2)
 {
+    // This alternate version may work better for platforms where acos() precision is low.
+    /*
+    float x = dot(v1, v2);
+    float y = abs(x);
+
+    float a = 5.42031 + (3.12829 + 0.0902326 * y) * y;
+    float b = 3.45068 + (4.18814 + y) * y;
+    float theta_sinTheta = a / b;
+
+    if (x < 0.0)
+    {
+        theta_sinTheta = PI * rsqrt(1.0 - x * x) - theta_sinTheta;
```

Comment:
Wondering if it would be better to saturate the input to rsqrt. I had a shadow bug recently where 1-dot(a,b) was occasionally a tiny bit under zero. When used as an input to sqrt it would produce nans

Reply:
This expects two normalized vectors so the dot should always be between -1 and 1 numerically, but I guess it's possible for floating point precision to cause issues? I'll add a saturate, it's basically free.